From 64e3e92fbb25407069cfc87c1f46f14a49661b29 Mon Sep 17 00:00:00 2001
From: Vihang Karajgaonkar <vihang@cloudera.com>
Date: Mon, 25 Jul 2016 14:16:36 -0700
Subject: [PATCH 0704/1164] CLOUDERA-BUILD : CDH-42576 : Fix bug in
 updateBeeLineOptsFromConf method in
 BeelineOpts.java due to old merge conflict

Change-Id: Ic3876a00a12e43aa74d62260cd39771094423d63
---
 .../java/org/apache/hive/beeline/BeeLineOpts.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/beeline/src/java/org/apache/hive/beeline/BeeLineOpts.java b/beeline/src/java/org/apache/hive/beeline/BeeLineOpts.java
index d5088ff..35acf91 100644
--- a/beeline/src/java/org/apache/hive/beeline/BeeLineOpts.java
+++ b/beeline/src/java/org/apache/hive/beeline/BeeLineOpts.java
@@ -230,7 +230,7 @@ public void load(InputStream fin) throws IOException {
   public void updateBeeLineOptsFromConf() {
     if (!beeLine.isBeeLine()) {
       if (conf == null) {
-        conf = beeLine.getCommands().getHiveConf(true);
+        conf = beeLine.getCommands().getHiveConf(false);
       }
       setForce(HiveConf.getBoolVar(conf, HiveConf.ConfVars.CLIIGNOREERRORS));
     }
-- 
1.7.9.5

